### PR TITLE
CCache: updated recipe, gcc2 not supported anymore.

### PR DESCRIPTION
**Pull request by miqlas**

### CODE REVIEW

File: dev-util/ccache/ccache-3.3.3.recipe [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Supported languages are C, C++, Objective-C and Objective-C++."
+HOMEPAGE="https://ccache.samba.org/"
+COPYRIGHT="2002-2007 Andrew Tridgell
+	2009-2016 Joel Rosdahl"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://samba.org/ftp/ccache/ccache-$portVersion.tar.bz2"
+CHECKSUM_SHA256="2985bc5e32ebe38d2958d508eb54ddcad39eed909489c0c2988035214597ca54"
+
+ARCHITECTURES="x86 x86_gcc2 x86_64"
```
Comment (korli) [inline dev-util/ccache/ccache-3.3.3.recipe]: Well, "gcc2 not supported anymore" isn't reflected here :)

**Comment by korli** [comment]
Well, I see now two problems: 
* cmd:ccache_x86 is installed on x86_gcc2 instead of cmd:ccache 
* the package ccache_x86 conflicts with the package ccache on x86_gcc2 (for people who installed a previous version, compatible with gcc2).
Something along the lines (to be tested)
`if [ "$targetArchitecture" = x86_gcc2 ]; then`
`       REPLACES="ccache"`
`fi`

**Comment by waddlesplash** [comment]
@miqlas: please address @korli's comments.

**Comment by waddlesplash** [comment]
> cmd:ccache_x86 is installed on x86_gcc2 instead of cmd:ccache

@korli: but GCC2 is not supported anymore, so shouldn't it stay this way?

**Comment by korli** [comment]
@waddlesplash x86_gcc2 should be marked as unsupported, and the x86 package should then provide cmd:ccache.

**Comment by scottmc** [comment]
Does this one build and work ok now?  Looks like gcc2 has been marked as not supported as suggested.

**Comment by scottmc** [comment]
Also, can a TEST() be added to this one?

**Comment by miqlas** [comment]
Ready.

**Pull request merged**